version.go

```diff
@@ -22,7 +22,9 @@
 
 package srp
 
+const Version string = "0.0.0"
```

Comment:
This also doesn't much matter, but maybe let's say that this is 0.0.1, and that the previous "undefined"(?!) version was 0.0.0, or something..
```suggestion
const Version string = "0.0.1"
```